doc/authoring_help.md

```diff
@@ -74,72 +74,6 @@ helps['account'] = """
             """
 </pre>
 
-
-### Example YAML help file, help.yaml (Version 1) ###
```

Comment:
Removed this since we don't want anyone using the pure YAML. It can be reverted if the team chooses to finish the help overhaul.